packages/foreman/nodejs-history/nodejs-history.spec

```diff
@@ -16,38 +19,49 @@ Source5: https://registry.npmjs.org/value-equal/-/value-equal-0.4.0.tgz
 Source6: https://registry.npmjs.org/resolve-pathname/-/resolve-pathname-2.2.0.tgz
 Source7: https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.13.2.tgz
 Source8: https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz
-Source9: %{name}-%{version}-registry.npmjs.org.tgz
-BuildRequires: nodejs-packaging
+Source9: nodejs-%{npm_name}-%{version}-registry.npmjs.org.tgz
```

Comment:
This looks incorrect and may also be the cause of its removal.

Reply:
Why does this look incorrect? (it works) In an SCL context, `name` now includes the prefix

Reply:
Probably something we'll find out in the npm2rpm process but we do rewrite some things. For example, `@theforeman/vendor` becomes `nodejs-theforeman-vendor`. Likely easier to fix in the template though.